sklearn/ensemble/_voting.py

```diff
@@ -64,13 +66,32 @@ def fit(self, X, y, sample_weight=None):
         """Get common fit operations."""
         names, clfs = self._validate_estimators()
 
+        check_scalar(
+            self.verbose,
+            name="verbose",
+            target_type=(numbers.Integral, np.bool_),
+            min_val=0,
+        )
+
         if self.weights is not None and len(self.weights) != len(self.estimators):
             raise ValueError(
                 "Number of `estimators` and weights must be equal"
                 "; got %d weights, %d estimators"
                 % (len(self.weights), len(self.estimators))
             )
 
+        if self.n_jobs is not None:
```

Comment:
For `n_jobs`, I had a question about `0` and whether is an acceptable value or not for this param. If it is acceptable, I can refactor this section to use `check_scalar`. I wrote out this simple validation as a placeholder in the meantime.

Reply:
`n_jobs=0` is invalid in joblib. I am pretty sure it raises an error here. Feel free to check.

Reply:
I agree that there is no need to add checks for `n_jobs` in scikit-learn because it is the responsibility of joblib to raise informative enough error messages.

Reply:
Thank you, okay, I removed all test and validations for this one. This was another comment saying the same thing: https://github.com/scikit-learn/scikit-learn/issues/20724#issuecomment-1016028188